### PR TITLE
Fix messages displayed for student and group commands

### LINKED ISSUE #183: [PE-D] No contextualisation for the target audience
In general, there are still traces of AB3 in the command results panel. Usage of `persons` instead of `students` and `address book` instead of `student list` etc.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: botr99/ped#9

### PR DESCRIPTION
**Pull request by Neha-5678**
Fixes #183. Edited the error messages and command usage messages for some student and group commands so that the correct messages appear on the UI (e.g. replaced person with student).

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/groups/DeleteGroupCommand.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
     public static final String MESSAGE_USAGE = GROUP_COMMAND + " " + COMMAND_WORD
             + ": Deletes the group identified by the index number used in the displayed group list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + "group " + COMMAND_WORD + " 1";
```
Comment (tanhuakun) [inline src/main/java/seedu/address/logic/commands/groups/DeleteGroupCommand.java:27]: You might want to use the GROUP_COMMAND message in [CliSyntax](https://github.com/AY2122S1-CS2103T-W16-4/tp/blob/6db7a2a3e7c77d44587b58d45c7bc26c387eedf4/src/main/java/seedu/address/logic/parser/CliSyntax.java).

File: src/main/java/seedu/address/logic/commands/groups/ViewGroupCommand.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
     public static final String MESSAGE_USAGE = GROUP_COMMAND + " " + COMMAND_WORD
             + ": Views a group in the groups list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + "group " + COMMAND_WORD + " 1";
```
Comment (tanhuakun) [inline src/main/java/seedu/address/logic/commands/groups/ViewGroupCommand.java:28]: Same here, see comment in DeleteGroupCommand

**Review by tanhuakun — APPROVED** [review]
LGTM! Take note that you can use the GROUP_COMMAND message in the future. Maybe change it in the next PR?

File: src/main/java/seedu/address/logic/commands/groups/DeleteGroupCommand.java (hunk 1 above)
Comment (Neha-5678) [reply src/main/java/seedu/address/logic/commands/groups/DeleteGroupCommand.java:27]: Oh yes, thanks for pointing that out!

**Pull request merged**